config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GetEnvVariable(key string) string {
+	err := godotenv.Load()
+	if err != nil {
```

Comment:
Consider using short syntax.

```suggestion
	if err := godotenv.Load(); err != nil {
```